### ISSUE TITLE
Radio buttons not operable using Firefox and JAWS 18

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#307]
User kind of got stuck on the first radio button.

Arrow keys did not work at accessing or activating the other 2 radio buttons.
User did get to the other 2, but very inconsistently and they were never able to activate the 2nd and 3rd radios.

Sim version 1.4.0-dev.4

**Comment by jessegreenberg** [comment]
@terracoda was the user used to using JAWS? We noticed that JAWS doesn't switch to forms mode like NVDA and VO for radio buttons in https://github.com/phetsims/a11y-research/issues/54, does it sound like something like that was happening?

**Comment by terracoda** [comment]
@jessegreenberg, yes, the user typically uses JAWS, and tried several times to activate the radio buttons.

**Comment by terracoda** [comment]
And, I did not hear the JAWS pop sound that indicates forms mode, so part of the problem may be the lack of forms mode. However, it seemed more than just a forms mode issue. It seemed like it was hard to move from radio button 1 to radio button 2 and 3.

**Comment by jessegreenberg** [comment]
Ok, thanks @terracoda I will test locally and see if I can reproduce.

**Comment by terracoda** [comment]
@jessegreenberg, today we tested the sim again with JAWS and FF. The user had the latest update for JAWS 18 professional (just got the update yesterday after our session).

It may be a coincidence, but we were able to activate the radio buttons this time.

You were right, radio buttons do not activate forms mode automatically. The user was able to use the Arrow keys to navigate to the 2nd and 3rd radio button, but they dd not auto-select. The user pressed Enter on the 3rd radio button (Show charge differences) and it was selected and JAWS made it's forms-mode pop. Then after that, moving the arrow keys to the previous radio buttons did auto-select the radio buttons.

I have no idea why JAWS behaves like this. But the radio buttons do work as implemented once the user activates forms mode.

As a result of exploring the radios and forms mode, we also discovered some quirks about exiting forms mode and entering application role which I will start a new issue for.

**Comment by jessegreenberg** [comment]
When I submitted a bug report to VFO about this, they responded with

> ...The radio button feature of JAWS is working correctly. When you navigate to a radio button, JAWS is not supposed to go into forms mode. You are to make your selection from the list of radio buttons with the spacebar. This checks the item, and allows you to move on the next control.

The description in https://github.com/phetsims/balloons-and-static-electricity/issues/307#issuecomment-332695464 does match this behavior though I also find it odd. But since this is as the JAWS devs intended it sounds like this is not really a bug. @terracoda do you agree and can this issue be closed?

**Comment by jessegreenberg** [comment]
It sounds like performance was worse in the last trial, wondering if JAWS was getting stuck in the previous test for that reason.

**Comment by terracoda** [comment]
Yes, I think we can close this one. I still think there is some weirdness about the radio buttons. We can test some more and see.

Closing for now.
